lib/codegen/CreateTerm.cpp

```diff
@@ -294,6 +293,16 @@ sptr<KORESort> termSort(KOREPattern *pattern) {
       fmt::format("{}:{}", locationStack, x))                                  \
       .first
 
+std::string escape(std::string str) {
+  std::stringstream os;
+  os << std::setfill('0') << std::setw(2) << std::hex;
+  for (char c : str) {
+    unsigned char uc = c;
+    os << (int)uc;
+  }
+  return os.str();
+}
+
```

Comment:
Had to manually resolve a conflict here by just choosing to include both versions (`ALLOC_ARG` is in master from someone else's changes, `escape` is something I'm reverting)
```
<<<<<<< HEAD                                                                                                                                                                                                                                  
#define ALLOC_ARG(x)                                                           \                                                                                                                                                              
  createAllocation(                                                            \                                                                                                                                                              
      pattern->getArguments()[x].get(),                                        \                                                                                                                                                              
      fmt::format("{}:{}", locationStack, x))                                  \                                                                                                                                                              
      .first                                                                                                                                                                                                                                  
=======                                                                                                                                                                                                                                       
std::string escape(std::string str) {                                                                                                                                                                                                         
  std::stringstream os;                                                                                                                                                                                                                       
  os << std::setfill('0') << std::setw(2) << std::hex;                                                                                                                                                                                        
  for (char c : str) {                                                                                                                                                                                                                        
    unsigned char uc = c;                                                                                                                                                                                                                     
    os << (int)uc;                                                                                                                                                                                                                            
  }                                                                                                                                                                                                                                           
  return os.str();                                                                                                                                                                                                                            
}                                                                                                                                                                                                                                             
>>>>>>> parent of 4c9d010 (Improve handling of Unicode vs Byte strings (#828))
```